apps/ospTutorial/ospTutorial.cpp

```diff
@@ -264,8 +266,10 @@ class Server
   {
     std::lock_guard<std::mutex> lock(_dataMutex);
     _webscene = webscene;
-    for (const auto &layer : _webscene.getFeatureLayers())
-      std::cout << layer << std::endl;
+    for (const auto &layerUrl : _webscene.getFeatureLayers()) {
+      std::cout << layerUrl << std::endl;
+      _loadPBF(*this, layerUrl);
```

Comment:
hier müsste eher ein `_loadLayer(layerUrl)` sein, das dann die `layerUrl` patched und wiederum `_loadPBF(*this, pbfUrl)` aufruft